docs/content/configuration.md

```diff
@@ -135,37 +137,39 @@ In some circumstances (e.g. when used within automated pipelines) it makes sense
+| `GRAFANA_TOKEN` | Basic auth password or API token.                     | false    | -         |
 
 See Grafana's [Authentication API
 docs](https://grafana.com/docs/grafana/latest/http_api/auth/) for more info.
 
 ## Grafana Cloud Prometheus
 To interact with Grafana Cloud Prometheus, you must have these environment variables set:
 
-| Name | Description | Required |
-| --- | --- | --- |
-| `CORTEX_ADDRESS` | URL for Grafana Cloud Prometheus instance | true |
-| `CORTEX_TENANT_ID` | Tenant ID for your Grafana Cloud Prometheus account | true |
-| `CORTEX_API_KEY` | Authentication token/api key | true |
+| Name               | Description                                         | Required |
+|--------------------|-----------------------------------------------------|----------|
+| `CORTEX_ADDRESS`   | URL for Grafana Cloud Prometheus instance           | true     |
+| `CORTEX_TENANT_ID` | Tenant ID for your Grafana Cloud Prometheus account | true     |
+| `CORTEX_API_KEY`   | Authentication token/api key                        | true     |
 
 Note, this will also work with other Cortex installations, alongside Grafana Cloud Prometheus.
 
 ## Grafana Synthetic Monitoring
 To interact with Grafana Synthetic Monitoring, you must have these environment variable set:
 
-| Name | Description | Required |
-| --- | --- | --- |
-| `GRAFANA_SM_TOKEN` | Authentication token/api key (must have MetricsPublisher permissions) | true |
-| `GRAFANA_SM_STACK_ID` | Grafana instance/stack ID | true |
-| `GRAFANA_SM_LOGS_ID` | Logs instance ID | true |
-| `GRAFANA_SM_METRICS_ID` | Metrics instance ID | true |
+| Name                    | Description                                                           | Required |
+|-------------------------|-----------------------------------------------------------------------|----------|
+| `GRAFANA_SM_TOKEN`      | Authentication token/api key (must have MetricsPublisher permissions) | true     |
+| `GRAFANA_SM_STACK_ID`   | Grafana instance/stack ID                                             | true     |
+| `GRAFANA_SM_LOGS_ID`    | Logs instance ID                                                      | true     |
+| `GRAFANA_SM_METRICS_ID` | Metrics instance ID                                                   | true     |
+| `GRAFANA_SM_REGION`     | Region of your instance                                               | false    |
 
 Your stack ID is the number at the end of the url when you view your Grafana instance details, ie. `grafana.com/orgs/myorg/stacks/123456` would be `123456`. Your metrics and logs ID's are the `User` when you view your Prometheus or Loki instance details in Grafana Cloud.
+You can check the regions under Synthetic Monitoring configuration. For example, if your backend URL is `synthetic-monitoring-api-gb-south.grafana.net`, your region should be `gb-south`. Default value is `synthetic-monitoring-api.grafana.net`.
```

Comment:
Is `synthetic-monitoring-api.grafana.net` the default? Or `us-central-1` or such?

Reply:
It's the value for `us`. My other comment will make that clearer